lib/plugins/babelRuntimePathResolver.js

```diff
@@ -2,11 +2,12 @@
 // in OBT's node_modules directory and not the module's
 'use strict';
 
-const path = require('path');
 const falafel = require('falafel');
 
 const prefix = /babel-runtime/;
-const rootDirectory = path.resolve(__dirname, '../../node_modules');
+
+// HACK: Can't require.resolve babel-runtime so use babel-core instead
+const rootDirectory = require.resolve('babel-core').replace(/(.*node_modules).*/, '$1');
```

Comment:
Thanks but I've realised mine is wrong :) If there's a browser field in the module being required, it will be silently ignored, because the require is rewritten to load the 'main' file directly. It would be better to use [browser-resolve](https://github.com/defunctzombie/node-browser-resolve) instead. But then we're reimplementing logic that webpack does. I think possibly the most sensible way is to use require.resolve but also chop off everything after the package's root (eg the `/lib/index.js` or whatever it is). Then we leave it up to webpack to look at the module's package.json and decide what to do.